src/setup/WixAdditionalTools/WixAdditionalTools.wxs

```diff
@@ -0,0 +1,27 @@
+<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information. -->
+
+<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs"
+     xmlns:bal="http://wixtoolset.org/schemas/v4/wxs/bal">
+  <Bundle Name="!(loc.Name)" Version="$(SetupVersion)" UpgradeCode="fc48131d-214a-439b-9a31-ef2ea5c69ea5"
+          AboutUrl="!(loc.AboutUrl)" HelpUrl="!(loc.SupportUrl)" UpdateUrl="!(loc.UpdateUrl)">
+    <BootstrapperApplication>
+      <bal:WixStandardBootstrapperApplication
+        LicenseUrl="!(loc.LicenseUrl)"
+        Theme="hyperlinkLicense"
+        ShowVersion="true"
+        LogoFile="wix-logo-sq.png" />
+    </BootstrapperApplication>
+
+    <SetVariable Variable="InstallFolder" Value="[ProgramFilesFolder]WiX Toolset v$(SetupMajorMinorVersion)\" />
```

Comment:
Shouldn't we be persisting this variable?

Reply:
Could do. But Windows Installer won't let us change the install location, so it doesn't have any effect. But it would probably look better in subsequent operation's log files if the value was the same.